pylintrc

```diff
@@ -72,6 +87,9 @@ min-similarity-lines=10
 # Ignore comments when computing similarities.
 ignore-comments=yes
 
+#we don't need to do pylint on the examples, too many false positives
```

Comment:
Style nit to match the pattern.
```suggestion
# We don't need to do pylint on the examples, too many false positives
```

Reply:
ooh well spotted.  I hadn't even noticed the highlighting was different in vscode. Fixed.